app/views/admin/orders/index.html.erb

```diff
@@ -1,5 +1,7 @@
 <h1>Admin Dashboard</h1>
```

Comment:
I feel like the admin is getting lots of special links. @CjMoore @meyerhoferc @valerietrudell maybe tomorrow we can ask about navigation best practices, it might make sense to do a dropdown or have a sidebar navigation that's special for admins.